### ISSUE TITLE
Error Running kotlin

### ISSUE THREAD

**Issue by Art1964** [formulahendry/vscode-code-runner#493]
- VS Code Version:1.35.0
- OS Version: Windows 10 pro
- Code Runner Version:0.9.10

**Describe the bug**
When executing the program kotlin the output screen shows the following message: "there is no main manifest attribute in test.jar"

**To Reproduce**

1. run hello world example
2. 

**Actual behavior**
Description of what happened actually.

**Expected behavior**
Description of what you expected to happen.

**Screenshots**

**Comment by Walker-ZHAO** [comment]
I meet the same issue!
VS Code Version: 1.3.51
OS Version: macOS 10.14.5
Code Runner Version: 0.9.11

**Comment by formulahendry** [comment]
Could you please share the screenshots?

**Comment by huqifan** [comment]
I think it's a kotlinc issue. Please use this command instead.
`
kotlinc hello.kt -include-runtime -d hello.jar && kotlin -classpath hello.jar HelloKt
`

**Comment by Walker-ZHAO** [comment]
> I think it's a kotlinc issue. Please use this command instead.
> `kotlinc hello.kt -include-runtime -d hello.jar && kotlin -classpath hello.jar HelloKt`

Yes, this command works properly, but due to it's not dynamic, I can't put it into my settings.json file,  I can use $fileName and $fileNameWithoutExt, but can't confirm the class name. The version of kotlinc is 1.3.40, look like it's a kotlinc issue, thx.

**Comment by fatfatson** [comment]
` @file:JvmName("HelloKt") `

use this to set class name

**Comment by whataa** [comment]
> `@file:JvmName("HelloKt")`
> 
> use this to set class name

Once successfully run once, it can run without any other settings. I don't know why.

**Comment by hassanahtasham** [comment]
I have the same problem, but above sol is not working
